lib/puppet/type/keycloak_realm.rb

```diff
@@ -186,4 +186,69 @@
     newvalues(:true, :false)
     defaultto :false
   end
+
+  newproperty(:smtp_server_user) do
+    desc 'smtpServer user'
+  end
+
+  newproperty(:smtp_server_password) do
+    desc 'smtpServer password'
+
+    def insync?(is)
+      if is =~ %r{^[\*]+$}
+        Puppet.warning("Property 'smtp_server_password' is set and Puppet has no way to check current value")
+        true
+      else
+        false
+      end
+    end
+
+    def should_to_s(_newvalue)
+      '[new smtp_server_password redacted]'
+    end
+  end
+
+  newproperty(:smtp_server_host) do
+    desc 'smtpServer host'
+  end
+
+  newproperty(:smtp_server_port) do
+    desc 'smtpServer port'
+    munge { |v| v.to_s }
+  end
```

Comment:
```suggestion
  newproperty(:smtp_server_port, parent: PuppetX::Keycloak::IntegerProperty) do
    desc 'smtpServer port'
  end
```

Reply:
This is proper way wit this module to make something an integer.

Reply:
Ok, thanks, will fix.